packages/ref-metrics-esmvaltool/src/ref_metrics_esmvaltool/example.py

```diff
@@ -0,0 +1,108 @@
+from typing import Any
+
+import xarray
+from ref_core.datasets import FacetFilter, SourceDatasetType
+from ref_core.metrics import DataRequirement, Metric, MetricExecutionDefinition, MetricResult
+from ruamel.yaml import YAML
```

Comment:
This is an aside, but why use ruamel over pyyaml?

Reply:
It supports roundtripping comments, which seems nice to have.